area.go

```diff
@@ -2,50 +2,161 @@ package cursor
 
 import (
 	"fmt"
+	"io"
 	"os"
+	"runtime"
 	"strings"
 )
 
 // Area displays content which can be updated on the fly.
 // You can use this to create live output, charts, dropdowns, etc.
 type Area struct {
-	height int
-	writer Writer
+	height     int
+	writer     io.Writer
+	cursor     *Cursor
+	cursorPosY int
 }
 
 // NewArea returns a new Area.
-func NewArea() Area {
-	return Area{
+func NewArea() *Area {
```

Comment:
Is there a reason that this is now returning a pointer?

Reply:
Reverted in latest commit, no change required. 
Can't even say why I did that in the first place ^^